tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs

```diff
@@ -102,10 +103,13 @@ internal static class DbScopeFactory
                     }
                     else
                     {
-                        var traceParentInjectedInComment = DatabaseMonitoringPropagator.PropagateDataViaComment(tracer.Settings.DbmPropagationMode, integrationId, command, tracer.DefaultServiceName, tagsFromConnectionString.DbName, tagsFromConnectionString.OutHost, scope.Span);
+                        var traceParentInjectedInComment = DatabaseMonitoringPropagator.PropagateDataViaComment(propagationMode, integrationId, command, tracer.DefaultServiceName, tagsFromConnectionString.DbName, tagsFromConnectionString.OutHost, scope.Span);
 
+                        var traceParentInjectedInContext = false;
```

Comment:
why do we need this ? The value is overriden just after 🤔

Reply:
yes merge conflict issue, abandoning this PR to just add into the other one